hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RSRpcServices.java

```diff
@@ -794,6 +792,29 @@ private Result increment(final HRegion region, final OperationQuota quota,
     return r == null ? Result.EMPTY_RESULT : r;
   }
 
+  private Get toGet(final Mutation mutation) throws IOException {
+    assert mutation instanceof Increment || mutation instanceof Append;
```

Comment:
Instead of `assert` statement, we can throw AssertionError with if condition? Just that assert might not be enabled server side.